### PR TITLE
windows test-net

### PR DESCRIPTION
**Pull request by J-Thompson12**
Closes: #XXX

## Description

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -74,7 +74,7 @@ docker:
 	docker cp ethermint:/usr/bin/emintcli ./build/
 
 docker-localnet:
-	# build the image
+# build the image
 	docker build -f ./networks/local/ethermintnode/Dockerfile . -t emintd/node
```
Comment (J-Thompson12) [inline Makefile:77]: i dont know if moving this comment will break other OS but for windows if the comment isnt moved over it tries to run it.

**Review by noot — APPROVED** [review]
works for me (on linux)!

File: Makefile [hunk 2]
```diff
@@ -74,7 +74,7 @@ docker:
 	docker cp ethermint:/usr/bin/emintcli ./build/
 
 docker-localnet:
-	# build the image
+# build the image
```
Comment (fedekunze) [inline Makefile]: we can just remove it
```suggestion
```

**Review by fedekunze — APPROVED** [review]
utACK

**Pull request merged**